### PR TITLE
Fix indication of the quota consumed

### PR DESCRIPTION
**Pull request by Jerome-Herbinet**
Signed-off-by: Jérôme Herbinet <33763786+Jerome-Herbinet@users.noreply.github.com>

Signed-off-by: Jérôme Herbinet <33763786+Jerome-Herbinet@users.noreply.github.com>

Before : 

After :

### CODE REVIEW

**Comment by susnux** [comment]
This does not work for me, for me it does the opposite: it breaks the quota bars. On NC 26 beta 2 this looks like this:

| Before (current master) | After (this PR) |
|---|---|
| quota with bars | quota not visible |

**Comment by Jerome-Herbinet** [comment]
> This does not work for me, for me it does the opposite: it breaks the quota bars. On NC 26 beta 2 this looks like this:
> Before (current master) 	After (this PR)
> quota with bars 	quota not visible

Hmmmm, strange, I've made some new test, and it's not breaking the UI. But I've noticed a missing property to let appear the background color of used quota (now fixed in a 2nd commit).

File: src/settings/EditSelect.scss [hunk 1]
```diff
@@ -19,22 +19,25 @@
...
+		border-radius: 10px;
+		overflow: hidden;
 	}
 
 	.quotabar {
 		position: absolute;
 		margin: 3px 0;
-		border-radius: 3px;
-		top: -7px;
+		border-radius: 0;
```
[the remark this answers is not in the document]
Comment (koelle25) [reply src/settings/EditSelect.scss:39]: Not directly needed to fix issue #2276, but a nice addition anyway. Without it the quotabar "overhangs" the select box a bit. I temporarily made the quotabar background red to better see it in this screenshot:


With `var(--border-radius-large)`:

**Review by koelle25 — CHANGES_REQUESTED** [review]
Hi Jerome,
thanks for trying to fix issue #2276!

There are a few changes in this PR that aren't actually needed and some changes would be better suited on other parts.
See my review :)

File: src/settings/EditSelect.scss [hunk 2]
```diff
@@ -19,22 +19,25 @@
 		height: 34px;
 		width: 150px;
 		position: absolute;
-		top: 0;
+		top: -3px;
```
[the remark this answers is not in the document]
Comment (Jerome-Herbinet) [reply src/settings/EditSelect.scss]: Done

File: src/settings/EditSelect.scss [hunk 3]
```diff
@@ -19,22 +19,25 @@
...
 		left: 0;
 		margin: 0;
+		background-color: transparent;
 	}
 
 	.quotabar-holder {
-		height: 34px;
+		height: 36px;
```
[the remark this answers is not in the document]
Comment (Jerome-Herbinet) [reply src/settings/EditSelect.scss]: Done

File: src/settings/EditSelect.scss [hunk 4]
```diff
@@ -19,22 +19,25 @@
...
 		width: 150px;
 		position: relative;
+		border-radius: 10px;
+		overflow: hidden;
```
[the remark this answers is not in the document]
Comment (Jerome-Herbinet) [reply src/settings/EditSelect.scss]: Done

File: src/settings/EditSelect.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jerome-Herbinet) [reply src/settings/EditSelect.scss]: Done

**Comment by Jerome-Herbinet** [comment]
> Hi Jerome, thanks for trying to fix issue #2276!
> 
> There are a few changes in this PR that aren't actually needed and some changes would be better suited on other parts. See my review :)

Hi @koelle25 ! 
Thank you very much for your review. 
I've followed all your suggestions.
I let you check if it's OK for you.
I remain at your disposal.
Bests regards,
Jérôme

**Comment by Jerome-Herbinet** [comment]
What it renders now 👍 @koelle25

**Comment by Jerome-Herbinet** [comment]
@susnux it should be OK now.

**Comment by Jerome-Herbinet** [comment]
@icewind1991 @juliushaertl this PR should make GF admin UI "great again" 😅

**Review by juliushaertl — APPROVED** [review]

**Comment by juliushaertl** [comment]
@Jerome-Herbinet Thanks for that. Could you maybe squash your commits into one?

**Comment by Jerome-Herbinet** [comment]
Will a "npm run sass" be necessary ?

**Comment by juliushaertl** [comment]
Not in this repo. Looks good now. Regarding squashing, a descriptive commit message would still be nice as this makes it easier to follow changes that happened later on. Maybe you can just give it a message like "Fix indication of the quota consumed" as the PR title.

**Comment by Jerome-Herbinet** [comment]
I've tried to add comment, but I'm not sure if it worked, can you confirm ?

**Comment by juliushaertl** [comment]
It is part of the commit message but the most important one would be the first line as that is what you usually see in git log and the GitHub web ui.

Maybe you can just replace the commit message with a fresh one with `git commit --amend`

**Comment by Jerome-Herbinet** [comment]
I think it's OK now.

**Comment by Jerome-Herbinet** [comment]
And thanks also @koelle25 :-)

**Comment by q-wertz** [comment]
After the update to 13.1.2 there now is an indicator but it still looks wrong…
(NC25.0.5)
image

**Comment by koelle25** [comment]
Yeah, it seems NC 25.0.5 brought another few changes which messed this up again :face_exhaling: On 25.0.4 it was working fine :sweat_smile:

**Comment by q-wertz** [comment]
@koelle25 Thanks for your help :)

Opened an issue. Regarding the color I had the problem that with `--color-background-dark` the text become unreadable so I switched it… :sweat_smile:

**Comment by Jerome-Herbinet** [comment]
> Yeah, it seems NC 25.0.5 brought another few changes which messed this up again face_exhaling On 25.0.4 it was working fine sweat_smile
> 
> Your proposal looks fine @q-wertz , though I maybe would rather use `background-color: var(--color-background-dark);` like it is used in the extended permissions field.
> 
> Separate issue I just noticed: The extended permissions field is not readable anymore due to a wrong text color it seems: 
> 
> I will open an issue for the second issue. @q-wertz I leave the honor of creating a new issue regarding the quota bar to you slightly_smiling_face

@koelle25 did you finally create an issue concerning ACL appearance ? If yes, I'd like to link my new PR to it (https://github.com/nextcloud/groupfolders/pull/2342)

**Comment by koelle25** [comment]
@Jerome-Herbinet Yes, it is here: #2331

**Pull request merged**